python/test/pyParticleEmitter_TEST.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+from ignition.math import Color, Pose3d, Vector3d
+from sdformat import ParticleEmitter
+import unittest
+
+
+class ParticleEmitterTEST(unittest.TestCase):
+
+
+    def test_default_construction(self):
+        emitter = ParticleEmitter()
+
+        self.assertFalse(emitter.name())
+
+        emitter.set_name("test_emitter")
+        self.assertEqual(emitter.name(), "test_emitter")
+
+        self.assertEqual("point", emitter.type_str())
+        self.assertEqual(ParticleEmitter.ParticleEmitterType.POINT, emitter.type())
+        self.assertTrue(emitter.set_type("box"))
+        self.assertEqual("box", emitter.type_str())
+        self.assertEqual(ParticleEmitter.ParticleEmitterType.BOX, emitter.type())
+        emitter.set_type(ParticleEmitter.ParticleEmitterType.CYLINDER)
+        self.assertEqual("cylinder", emitter.type_str())
+
+        self.assertTrue(emitter.emitting())
+        emitter.set_emitting(False)
+        self.assertFalse(emitter.emitting())
+
+        self.assertAlmostEqual(0.0, emitter.duration())
+        emitter.set_duration(10.0)
+        self.assertAlmostEqual(10.0, emitter.duration())
+
+        self.assertAlmostEqual(5.0, emitter.lifetime())
+        emitter.set_lifetime(22.0)
+        self.assertAlmostEqual(22.0, emitter.lifetime())
+        emitter.set_lifetime(-1.0)
```

Comment:
The C++ code has `EXPECT_DOUBLE_EQ(gz::math::MIN_D, emitter.Lifetime());`. Do 
we need to add that?

Reply:
https://github.com/gazebosim/sdformat/pull/999/commits/27a4ae4369eb9031c85655ee19b7c0d34b17e199